### PR TITLE
solution

### PR DESCRIPTION
**Pull request by Andrey1104**

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -16,4 +16,11 @@
...
 
-# write your code here
+sorted_variables = {"immutable": [lucky_number,
+                                  pi,
+                                  one_is_a_prime_number,
+                                  name,
+                                  profile_info],
+                    "mutable": [my_favourite_films,
+                                marks,
+                                collection_of_coins]}
```
Comment (viktoria-rybenchuk) [inline app/main.py:26]: Consider putting each key with its corresponding value on a new line for better readability.

**Review by viktoria-rybenchuk — CHANGES_REQUESTED** [review]

File: .flake8 [hunk 2]
```diff
@@ -1,7 +1,7 @@
 [flake8]
 inline-quotes = "
 ignore = E203, E266, W503, ANN002, ANN003, ANN101, ANN102, ANN401, N807, N818
-max-line-length = 79
+max-line-length = 96
```
Comment (Polyakiv-Andrey) [inline .flake8:4]: In this file there are flake8 rules that Python developers should follow, if you made a mistake you need to fix your code not this file, return max-line-length=79

File: app/main.py [hunk 3]
```diff
@@ -16,4 +16,6 @@
 }
 collection_of_coins = {1, 2, 25}
 
-# write your code here
+sorted_variables = {"immutable": [lucky_number, pi, one_is_a_prime_number, name, profile_info],
```
Comment (Polyakiv-Andrey) [inline app/main.py:19]: Correct your code according to the following structure
Screenshot 2023-10-16 at 16 58 49

**Review by Polyakiv-Andrey — CHANGES_REQUESTED** [review]

**Review by Polyakiv-Andrey — APPROVED** [review]
Good job!

**Pull request closed without merging**